SharePointFramework/PortfolioWebParts/src/models/ProjectListModel.ts

```diff
@@ -28,6 +31,9 @@ export class ProjectListModel {
     public groupId: string,
     public title: string,
     public url: string,
+    public GtProjectLifecycleStatus: string, 
+    public GtProjectServiceAreaText: string,
```

Comment:
Don't use the full field name.

Use `serviceArea`.

Reply:
Done ✅